opencensus/OpenCensus-Traces.md

```diff
@@ -0,0 +1,28 @@
+# OpenCensus Trace Exporters
+
+## Exporting Spans
+
+Each span received by the OpenCensus Exporter must be transformed into a New Relic span before being sent.
+
+|NR Span Field|OpenCensus Span Field|Type|Required|Comments|
+|-----|-----|-----|-----|-----|
+|`id`|`Span.SpanID`|string|yes|Unique identifier for this span|
+|`trace.id`|`Span.SpanContext.TraceID`|string|yes|Unique identifier shared by all spans within a single trace|
+|`name`|`Span.Name`|string|yes|The name of this span|
+|`parent.id`|`Span.ParentSpanID`|string|yes (except root span)|The span id of the previous caller of this span. Can be empty if this is the first span in which case the key should be omitted.|
+|`timestamp`|`Span.StartTime`|long|yes|Epoch ms timestamp|
+|`duration.ms`|`Span.EndTime` - `Span.StartTime`|float|yes|Duration of this span in milliseconds|
+|`service.name`|(none)|string|yes| The name of the service that created this span. This value is not gathered from OpenCensus. Instead it should be set directly by the user when defining the Exporter.|
```

Comment:
`name`, `parent.id`, `duration.ms` and `service.name` are all in the `attributes` rather than the top level. Should that be called out in here?